handbook/sales/README.md

```diff
@@ -143,6 +143,12 @@ To close a deal with a new customer (non-self-service), create and complete a Gi
 ### Change customer credit card number
 You can help a Premium license dispenser customers change their credit card by directing them to their [account dashboard](https://fleetdm.com/customers/dashboard). On that page, the customer can update their billing card by clicking the pencil icon next to their billing information.
 
+### Process a security questionnaire
+- The AE will use the handbook to answer most of the questions with links to appropriate sections in the handbook. After this first pass has been completed, and if there are outstanding questions, the AE will assign the issue to Business Operations (BizOps) (#g-business-operations) with a requested timeline for completion defined.
```

Comment:
@dherder Let's link to those sections here some how

Reply:
How will they assign it to BizOps? Probably issue, I would just send them to the "Contact-us" section on the Bizops page here

Reply:
I would also spell out that the AE is responsible for keeping track of this the whole way through. That way we have a clear dri until it's passed off to an SC.

Reply:
done